components/formats-gpl/src/loci/formats/in/DeltavisionReader.java

```diff
@@ -161,6 +162,11 @@ public boolean isThisType(String name, boolean open) {
   /* @see loci.formats.IFormatReader#isThisType(RandomAccessInputStream) */
   @Override
   public boolean isThisType(RandomAccessInputStream stream) throws IOException {
+    // Extra sanity check; if it's a TIFF, return false immediately, to avoid
+    // false negatives.
+    if(new TiffParser(stream).isValidHeader()) {
+      return false;
+    }
```

Comment:
I'm still not 100% convinced that this can't be replaced by a further check on ```x * y * count``` later on, which is negative in this case due to long overflow.  Perhaps wrapping each variable in a ```BigInteger``` instead of casting to ```long``` before multiplying would help and allow this to be removed?